e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-to-an-existing-payment.spec.js

```diff
@@ -149,4 +153,80 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can add fee records to existing paymen
       pages.utilisationReportAddToAnExistingPaymentPage.errorSummary().contains('Select a payment to add the fee or fees to');
     });
   });
+
+  it('should add the selected fees to the payments and update the fee record status after clicking the continue button', () => {
+    cy.task(NODE_TASKS.REMOVE_ALL_UTILISATION_REPORTS_FROM_DB);
+    cy.task(NODE_TASKS.REMOVE_ALL_PAYMENTS_FROM_DB);
+    cy.task(NODE_TASKS.INSERT_UTILISATION_REPORTS_INTO_DB, [report]);
+    cy.task(NODE_TASKS.INSERT_FEE_RECORDS_INTO_DB, [firstFeeRecord, secondFeeRecord]);
+
+    navigateToAddToExistingPaymentScreenForFirstFeeRecord();
+
+    pages.utilisationReportAddToAnExistingPaymentPage.continueButton().click();
+
+    pages.utilisationReportPage.premiumPaymentsTab.getPaymentLink(PAYMENT_ID_ONE).should('contain', 'GBP 450.00');
+    pages.utilisationReportPage.premiumPaymentsTab.getPremiumPaymentsTableRow(FEE_RECORD_ID_ONE).should('contain', 'MATCH');
+  });
```

Comment:
Are we able to use "equals" assertions here instead of `contain`?

I have a PR that introduces an `assertText` command to make this easier, but for now could use invoke and trim:

```js
theSelector
  .invoke('text')
  .then((text) => {
    expect(text.trim()).to.equal('MATCH');
  });
```

Reply:
`getPremiumPaymentsTableRow` returns all the cells in the row so we wouldn't be able to apply it to that.

We could add a `data-cy` attribute to the individual cell and introduce a new cypress helper `getPremiumPaymentsTableRowStatus` to get this cell value maybe?

Reply:
Sounds good, could do a few things, not sure how big of a change this would be, maybe should be a different PR, I think we should go with something like this:

```js
// DTFS
row: {
  exporter: (id) => cy.get(`[data-cy="deal__exporter--${id}"]`),
  bankRef: (id) => cy.get(`[data-cy="deal__bankRef--bankInternalRefName${id}"]`),
}

// EXIP
row: (referenceNumber) => ({
  status: () => cy.get(`[data-cy="ref-${referenceNumber}-status"]`),
  buyerLocation: () => cy.get(`[data-cy="ref-${referenceNumber}-buyerLocation"]`),
})

// So eventually we can do eg...
// cy.assertText(table.body.row.exporter(dealId) 'Example');
// cy.assertText(table.body.row(referenceNumber).status, IN_PROGRESS);
```